site/docs/v1/tech/guides/advanced-registration-forms.adoc

```diff
@@ -0,0 +1,540 @@
+---
+layout: doc
+title: Advanced Registration Forms
+description: Advanced Registration Forms
+---
+:page-liquid:
+
+== What Are Advanced Registration Forms
+
+Advanced registration forms let you build powerful, multi-step, custom registration experiences with no coding required.
+
+image::guides/advanced-registration-forms/built-out-advanced-form.png[Example of built out advanced registration form.,width=1200,role=box-shadow margin-right]
+
+You might be interested in this feature if you use the FusionAuth themed login pages for your application, and the default self service registration form doesn't meet your needs. Whether you want to break a form up into multiple steps for a better user experience, gather user consents, or have the user provide app specific data, advanced registration forms can help.
+
+If you are building your own login and registration pages using the link:../apis[APIs], you can still use the form builder in the administrative user interface, but you will have to generate the user facing HTML from the configured form data and recreate any front end logic. 
+
+== How Do I Use Advanced Registration Forms?
+
+*This is a paid edition feature and is unavailable in the community edition. Please visit link:/pricing[our pricing page] to learn more about paid editions.*
```

Comment:
Should we use a positive instead of a negative here? 

> This is feature is available in all paid editions of FusionAuth.

Reply:
Sure, makes sense.